pkgs/top-level/php-packages.nix

```diff
@@ -191,7 +194,7 @@ lib.makeScope pkgs.newScope (self: with self; {
 
     php-parallel-lint = callPackage ../development/php-packages/php-parallel-lint { };
 
-    phpcbf = callPackage ../development/php-packages/phpcbf { };
+    phpcbf = throw "phpcbf is included in phpcs, use phpcs instead.";
```

Comment:
Might be nice to hide it with `} // lib.optionalAttrs config.allowAliases {`

Reply:
fixed.